platform/javascript/display_server_javascript.cpp

```diff
@@ -131,8 +131,8 @@ Ref<InputEventKey> DisplayServerJavaScript::setup_key_event(const EmscriptenKeyb
 	ev.instantiate();
 	ev->set_echo(emscripten_event->repeat);
 	dom2godot_mod(emscripten_event, ev);
-	ev->set_keycode(dom_code2godot_scancode(emscripten_event->code, emscripten_event->key, false));
-	ev->set_physical_keycode(dom_code2godot_scancode(emscripten_event->code, emscripten_event->key, true));
+	ev->set_keycode((Key)dom_code2godot_scancode(emscripten_event->code, emscripten_event->key, false));
+	ev->set_physical_keycode((Key)dom_code2godot_scancode(emscripten_event->code, emscripten_event->key, true));
```

Comment:
I think `dom_code2godot_scancode` should return a `Key` instead given the name (in `platform/javascript/dom_keys.inc`).